<launch>
    <node pkg="demo_reconfigure_user" type="pretty_solve_user" name="pretty_solve_user" output="screen" >
        <param name="Navigation_Mode" type="bool" value="true" />
        <param name="Patrol_Mode" type="bool" value="false" />
        <param name="Rotate_Mode" type="bool" value="false" />
        <param name="Counterattack_Mode" type="bool" value="false" />
        <param name="waypointXYRadius" type="double" value="0.5" />
        <param name="waypointZBound" type="double" value="5.0" />
        <param name="waitTime" type="double" value="0" />
        <param name="frameRate" type="double" value="5.0" />
        <param name="speed" type="double" value="2.0" />
        <param name="sendSpeed" type="bool" value="true" />
    </node>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />
</launch>